src/ucp/core/ucp_rkey.c

```diff
@@ -572,13 +572,11 @@ void ucp_rkey_destroy(ucp_rkey_h rkey)
     }
 }
 
-ucp_lane_index_t ucp_rkey_find_rma_lane(ucp_context_h context,
-                                        const ucp_ep_config_t *config,
-                                        ucs_memory_type_t mem_type,
-                                        const ucp_lane_index_t *lanes,
-                                        ucp_rkey_h rkey,
-                                        ucp_lane_map_t ignore,
-                                        uct_rkey_t *uct_rkey_p)
+ucp_lane_index_t
+ucp_rkey_find_rma_lane(ucp_context_h context, const ucp_ep_config_t *config,
+                       ucs_memory_type_t mem_type,
+                       const ucp_lane_index_t *lanes, ucp_rkey_h rkey,
+                       ucp_lane_map_t ignore, uct_rkey_t *uct_rkey_p)
```

Comment:
minor: I'd avoid unrelated changes. This is the only change in this file and it is not relevant to the feature